templates/index

```diff
@@ -145,6 +145,7 @@ cisco_ios_show_version.template, .*, cisco_ios, sh[[ow]] ver[[sion]]
 cisco_ios_show_ip_arp.template, .*, cisco_ios, sh[[ow]] i[[p]] a[[rp]]
 cisco_ios_show_ip_bgp.template, .*, cisco_ios, sh[[ow]] i[[p]] bgp
 cisco_ios_show_clock.template, .*, cisco_ios, sh[[ow]] clo[[ck]]
+cisco_ios_show_tacacs.template, .*, cisco_ios, sh[[ow]] tacacs
```

Comment:
what are the minimal characters required after `sh[[ow]]` to get tacacs command to execute?

Reply:
is `sh tac` so I believe should be sh[[ow]] tac[[acs]]

Reply:
that would be correct then